### DISCUSSION TITLE
Hide field in Edit Mode

### DISCUSSION THREAD

**Discussion by elenaaralla** [serenity-is/Serenity#6848] [General]
Hi,

I would like to hide a property when a xyzRow in in edit mode; see image for details.

How can i do this? Is there an example in Demo or documentation?

Thank you,
Elena.

![Schermata 2023-07-21 alle 10 06 28](https://github.com/serenity-is/Serenity/assets/14870650/d1044bc7-d343-455c-ab63-9da0ae3fa076)

**Comment by ShifanilaSenthil** [comment]
Use this
If(this.Editmode()) {
This. Form. YOURFIELD. element. Hide() ;
}

**Reply by elenaaralla** [reply]
Hi,

The editor display an error "The property Hide() does not exist on Type LoookupTable) :(

some other idea?

**Reply by luongnv6** [reply]
If(this.Editmode()) {
 this.form.UserId.getGridField().hide();
}

**Reply by elenaaralla** [reply]
Hi @luongnv6,

with your suggested code, the UserId LookUp was hidden in both _Insert Customer_ and _Edit customer_

I would like that the lookup was visible ONLY when I **create new** customer.

Thank you,
Elena.

**Reply by elenaaralla** [reply]
Hi @luongnv6,

sorry, I forgot the parentheses to isEditMode :(

It works! Thank you!
